esp-bootloader-esp-idf/src/fmt.rs

```diff
@@ -0,0 +1,226 @@
+#![macro_use]
+#![allow(unused_macros)]
+
+#[cfg(all(feature = "defmt", feature = "log"))]
+compile_error!("You may not enable both `defmt` and `log` features.");
+
+macro_rules! assert {
```

Comment:
Please paste the up to date version of this, with collapse_debuginfo on the macros

Reply:
I thought I did ... apparently not